resources/dicts/thoughts/alive/general.json

```diff
@@ -51,87 +49,1497 @@
+            "Spoke out of turn at the last Clan meeting",
+            "Bravely offers to help the medicine cat pick herbs from a dangerous spot",
+            "Runs out of {PRONOUN/m_c/poss} den into 'battle', only to find that it was just a storm",
+            "Prances around with {PRONOUN/m_c/poss} tail up high, ready for anything",
+            "Promises the kits {PRONOUN/m_c/subject}'ll protect them when the time comes",
+            "Is not afraid of anything",
+            "Is always first to volunteer when a dangerous mission is announced"
+        ],
+        "main_trait_constraint": [
+            "bold"
+        ]
+    },
+    {
+        "id": "calm_gen",
+        "thoughts": [
+            "Is taking a moment to appreciate some peace and quiet",
+            "Is patiently listening to someone talk about things they are unquestionably wrong about and dying inside",
+            "Looks on as the camp buzzes around {PRONOUN/m_c/object}",
+            "Is basking in the gentle sunlight around camp",
+            "Is meditating upon a sun-warmed rock",
+            "Is sitting quietly on the far side of camp",
+            "Is humming a soothing tune to {PRONOUN/m_c/self} as {PRONOUN/m_c/subject} {VERB/m_c/work/works}",
+            "Is out on a peaceful stroll",
+            "Is remembering simpler days with fondness",
+            "Is offering gentle advice to a frustrated Clanmate",
+            "Is lining {PRONOUN/m_c/poss} nest with lavender",
+            "Smells sweet, like lavender",
+            "Never fails to offer comfort to {PRONOUN/m_c/poss} Clanmates",
+            "Is taking {PRONOUN/m_c/poss} sweet time eating {PRONOUN/m_c/poss} meal",
+            "Is purring quietly",
+            "Has been reaching meditative states while sunbathing",
+            "Calmly takes the lead while some Clanmates argue over this and that",
+            "Knows everything will be okay in the end",
+            "Takes a deep breath and doesn't let a mean comment get to {PRONOUN/m_c/object}"
+        ],
+        "main_trait_constraint": [
+            "calm"
+        ]
+    },
+    {
+        "id": "careful_gen",
+        "thoughts": [
+            "Is double-checking {PRONOUN/m_c/poss} nest for burrs",
+            "Is getting a small scratch checked out by the medicine cat",
+            "Is slowly and methodically grooming {PRONOUN/m_c/self}",
+            "Is chiding a younger cat for being so reckless",
+            "Is padding back and forth, across the camp",
+            "Is helping to reinforce the nursery walls",
+            "Is helping to reinforce the camp walls",
+            "Is patching up a hole in the camp wall",
+            "Is helping to reinforce the walls of the elders' den",
+            "Fortifies the camp walls for the fourth time this week",
+            "Feels at peace"
+        ],
+        "main_trait_constraint": [
+            "careful"
+        ]
+    },
+    {
+        "id": "charismatic_gen",
```

Comment:
separate out the thoughts that mention another warrior/s into their own thought group that requires a random warrior

Reply:
Either would be fine